css/style.css

```diff
@@ -0,0 +1,460 @@
+  background-color: white;
+}
+
+.header-middle::before {
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: "";
+  display: block;
+  width: 70px;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+.header-middle_links {
+  width: 88%;
+}
+
+.header-middle_links li {
+  border-right: 1px solid rgb(219, 209, 209);
+}
+
+.header-middle_links a {
+  padding: 0 calc(1.7vw - 10px);
+  color: black;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: 700;
+  font-size: 19px;
+  white-space: nowrap;
+}
+
+.header-middle_search > p {
+  font-family: "Roboto Condensed", sans-serif;
+  font-size: 19px;
+}
+
+.header-middle_search > img {
+  height: 25px;
+  width: 25px;
+}
+
+.header-bottom {
+  height: 43px;
+  background-color: white;
+  border-top: 2px #f4f5f6 solid;
+  border-bottom: 2px #f4f5f6 solid;
+}
+
+.header-bottom a {
+  margin: 0 1rem;
+  font-weight: 700;
+}
+
+.header-middle_links a:hover {
+  color: #f72210;
+  text-decoration: none;
+}
+
+@media screen and (max-width: 767px) {
+  .fixednav-btns > * {
+    margin: 0 calc(1vw - 10px) !important;
+  }
+
+  .fixednav-btn-subsc,
+  .fixednav-btn-toggle {
+    transform: scale(0.7);
+  }
+}
+
+/* Main section starts here */
+
+.main-container {
+  background-color: #fff;
+  padding-right: 0 !important;
+  height: auto !important;
+}
+
+.main-news {
+  width: 100%;
+}
+
+.main-news-col-1 .card {
+  margin: 0 0 0.8rem 0;
+}
+
+.main-news-col-1 .card-custom-img {
+  position: relative;
+  width: 100%;
+  max-height: 260px;
+  overflow: hidden;
+}
+
+.card-img-overlay {
+  top: unset;
+  bottom: 0;
+  left: 0;
+  right: unset;
+  display: inline-block;
+  width: auto;
+  height: 35px;
+  padding: 0 1rem;
+  line-height: 40px;
+  text-align: center;
+  font-size: 17px;
+  font-weight: 700;
+  color: #f72210;
+  background-color: rgba(240, 255, 240, 0.966);
+}
+
+.main-news-col-1 .card-title,
+.main-news-col-2 .card-title {
+  margin: 0.6rem 0 0 0;
+  font-size: 24px;
+  text-decoration: none;
+}
+
+.main-news-col-1 .card-text,
+.main-news-col-2 .card-text {
+  font-size: 17px;
+}
+
+.main-news-col-2 {
+  width: 42%;
+}
+
+.main-news-col-2 .card-custom-img {
+  position: relative;
+  width: 100%;
+  max-height: 310px;
+  overflow: hidden;
+}
+
+.news-grid .card-custom-img {
+  height: 160px;
+}
+
+.news-grid img {
+  height: 100%;
+}
+
+.news-grid .card-title,
+.news-grid-mig {
+  font-size: 16px;
+  font-weight: 600;
+  margin: 0.7rem 0;
+}
+
+.news-grid .card-title a,
+.news-grid-mig .card-title a {
+  color: rgba(3, 3, 3, 0.4) !important;
+}
+
+.news-grid .card-img-overlay {
+  height: 25px;
+  font-size: 14px;
+  line-height: 25px;
+  color: black;
+  background-color: rgba(206, 207, 206, 0.966);
+}
+
+.news-stories {
+  height: 1050px;
+  overflow-y: auto;
+}
+
+.news-stories-container {
+  min-height: 110px !important;
+  border-top: 1px solid var(--color2);
+}
+
+.news-stories-avatar {
+  height: 95px !important;
+}
```

Comment:
## Using the `!important` Rule[1]
>The `!important` the rule was conscripted in the mid-to-late ’90s to help web designers and developers override the rules of their style sheet. The `!important` rule prevails over all the previous styles and says, “I’m more important, forget about all of that other CSS and use me!“. 
 - [ ] Remove all your `!important` rules on the entire page, not just this section.
<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
I have removed all !important rules. Thank You.